episodes/1-introduction.Rmd

```diff
@@ -7,24 +7,26 @@ exercises: 30
 :::::::::::::::::::::::::::::::::::::: questions
 
 - What is deep learning?
+- What is a neural network?
+- Which operations are performed by a single neuron?
+- How do neural networks learn?
 - When does it make sense to use and not use deep learning?
-- When is it successful?
-- What are the tools involved?
+- What are tools involved in deep learning?
 - What is the workflow for deep learning?
 - Why did we choose to use Keras in this lesson?
-- How do neural networks learn?
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 :::::::::::::::::::::::::::::::::::::: objectives
 
+- Define deep learning
+- Describe how a neural network is build up
+- Explain the operations performed by a single neuron
+- Describe what a loss function is
```

Comment:
Minor remark, as we are re-thinking objectives: are these really objectives? I would see it rather along these lines:
```suggestion
- Lear the definition of deep learning
- Understand how a neural network is build up
- Learn about the operations performed by a single neuron
- Understand what a loss function is
```
Anyway, I think it is fine as it is. I just wanted to share my thoughts on the question of what an objective really is.

Reply:
This is on purpose 😂 Carpentries-style. Starts with verbs from the Bloom taxonomy. But I agree it seems weird if you don't know about it.